### PR TITLE
Add `renzoOracle()` getter

### PR DESCRIPTION
**Pull request by sentilesdal**
- return immutable value
- add renzoOracle getter

### CODE REVIEW

File: contracts/src/instances/ezeth/EzETHTarget0.sol [hunk 1]
```diff
@@ -37,9 +37,15 @@ contract EzETHTarget0 is HyperdriveTarget0, EzETHBase {
     }
 
     /// @notice Gets the ezETH token contract.
-    /// @return The ezETH token contract.
+    /// @return IERC20 The ezETH token contract.
```
Comment (jalextowle) [inline contracts/src/instances/ezeth/EzETHTarget0.sol:40]: IERC20 is a type and shouldn’t be in the NatSpec.

File: contracts/src/instances/ezeth/EzETHTarget0.sol [hunk 2]
```diff
@@ -37,9 +37,15 @@ contract EzETHTarget0 is HyperdriveTarget0, EzETHBase {
...
     function ezETH() external view returns (IERC20) {
-        _revert(abi.encode(_restakeManager.ezETH()));
+        _revert(abi.encode(_ezETH));
+    }
+
+    /// @notice Gets the Renzo Oracle contract.
+    /// @return IRenzoOracle The RenzoOracle contract.
```
Comment (jalextowle) [inline contracts/src/instances/ezeth/EzETHTarget0.sol:46]: Same here

File: contracts/src/instances/ezeth/EzETHTarget0.sol (hunk 1 above)
Comment (sentilesdal) [reply contracts/src/instances/ezeth/EzETHTarget0.sol:40]: should this be `address` instead?

Comment (jalextowle) [reply contracts/src/instances/ezeth/EzETHTarget0.sol:40]: If you check out other comments in the repo, you'll see what I mean. There isn't a type in any other `@return` statement.

File: contracts/src/instances/ezeth/EzETHTarget0.sol [hunk 3]
```diff
@@ -31,15 +31,21 @@ contract EzETHTarget0 is HyperdriveTarget0, EzETHBase {
...
 
     /// @notice Gets the ezETH token contract.
     /// @return The ezETH token contract.
     function ezETH() external view returns (IERC20) {
-        _revert(abi.encode(_restakeManager.ezETH()));
+        _revert(abi.encode(_ezETH));
+    }
+
+    /// @notice Gets the Renzo Oracle contract.
+    /// @return The RenzoOracle contract.
```
Comment (jalextowle) [inline contracts/src/instances/ezeth/EzETHTarget0.sol:46]: Why is it `RenzoOracle` here and `Renzo Oracle` above? Let's pick one?

File: contracts/src/interfaces/IEzETHHyperdriveRead.sol [hunk 4]
```diff
@@ -13,4 +13,8 @@ interface IEzETHHyperdriveRead is IHyperdriveRead {
     /// @notice Gets the ezETH token contract.
     /// @return The ezETH token contract.
     function ezETH() external view returns (IERC20);
+
+    /// @notice Gets the Renzo Oracle contract.
+    /// @return The RenzoOracle contract.
```
Comment (jalextowle) [inline contracts/src/interfaces/IEzETHHyperdriveRead.sol:18]: Same here.

**Pull request merged**